### DISCUSSION TITLE
Geomap not zooming to data

### DISCUSSION THREAD

**Discussion by anoppe-luminis** [teslamate-org/teslamate#4029] [Q&A] [answered]
I might be overlooking something, but I want to check with you before logging a bug: 
I'm running Teslamate since the beginning of June when I got my Tesla. However, like a week ago the geo panels in Grafana don't zoom to data anymore, for example:
![image](https://github.com/teslamate-org/teslamate/assets/173072849/2c0c61f3-1efb-4f0a-a5d6-198f05b5acb9)

Did a bug sneak into the software somehow, or is it config related?

**Comment by DrMichael** [comment]
Which dashboard is this? Apparently there is no data for the map...

**Reply by anoppe-luminis** [reply]
oh, there is data, let me zoom in a bit for you:
![image](https://github.com/teslamate-org/teslamate/assets/173072849/03c0f256-1d3d-460f-a449-7c29a5606501)
This example is from the 'Drives' dashboard, but all others with a Geomap show the same behavior.

**Reply by DrMichael** [reply]
Interesting. My Drives dashboard does not show a map?!?

**Reply by anoppe-luminis** [reply]
oh, my bad: Drive _details_ ... 🙃

**Comment by DrMichael** [comment, marked as answer]
Right, bug in Grafana Geomap. Will be fixed in one of the next releases: https://github.com/grafana/grafana/pull/89248

**Reply by anoppe-luminis** [reply]
cool, thank you for pointing this out!
